libs/auth/src/certs.ts

```diff
@@ -0,0 +1,111 @@
+import { Buffer } from 'buffer';
```

Comment:
This first commit isn't adding any new code. It's just moving code around and reorganizing helpers. As noted in the PR description and commit message, the breakdown of files is:

- `openssl.ts `- Generic openssl commands (not VotingWorks-specific)
- `piv.ts` - PIV standards (not VotingWorks-specific)
- `certs.ts` - VotingWorks-specific cert logic
- `java_card.ts` - Top-level logic